tutorials/python/python-video-generation/code/python_video.py

```diff
@@ -0,0 +1,52 @@
+import moviepy.editor as mpy
+import gizeh as gz
+
+
+def render_text(t):
+    BLUE = (59/255, 89/255, 152/255)
```

Comment:
@Larox I see `BLUE` is a constant. Does sound good to you if we move all constants out of function scope and group them at the top of the script?
```python
BLUE = ...
WHITE = ...
VIDEO_SIZE = ...
...

def render_text(text):
    ....


...
```